chaos_monkey/config.py

```diff
@@ -204,17 +204,13 @@ def as_dict(self):
     ms08_067_remote_user_add = "Monkey_IUSER_SUPPORT"
     ms08_067_remote_user_pass = "Password1!"
 
-    # psexec exploiter
-    psexec_user = "Administrator"
-    psexec_passwords = ["Password1!", "1234", "password", "12345678"]
-
-    # ssh exploiter
-    ssh_users = ["root", 'user']
-    ssh_passwords = ["Password1!", "1234", "password", "12345678"]
-
     # rdp exploiter
     rdp_use_vbs_download = True
 
+    # User and password dictionaries for exploits.
+    exploit_user_list = []
+    exploit_password_list = []
+
```

Comment:
Add in a properties to generate user/password pairs, duplicate code in rdpgrinder, smbexec, etc.